rpm/gratia-probe.spec

```diff
@@ -239,6 +239,7 @@ git_commit_id=$(gzip -d < %{SOURCE0} | git get-tar-commit-id)
   # Install the htcondor-ce configuration
   install -d $RPM_BUILD_ROOT/%{_sysconfdir}/condor-ce/config.d
   install -m 644 condor/99_gratia-ce.conf $RPM_BUILD_ROOT/%{_sysconfdir}/condor-ce/config.d/99_gratia-ce.conf
+  install -d $RPM_BUILD_ROOT/%{_sharedstatedir}/condor-ce/gratia/data
```

Comment:
You should change the update_probeconfig line (265) to use `%{_sharedstatedir}`, for consistency.

Reply:
Good catch, done